.gitignore

```diff
@@ -4,3 +4,10 @@
 # Build artifacts
 src/build/
 /bazel-*
+
+# Tulsi files
+*.tulsiconf-user
+tulsi-*
+
+# Xcode project
+/Tulsi/
```

Comment:
Should this be `/Tulsi.xcodeproj/`?

Reply:
Also, I'm going to hand off any further reviewing to @DavidGoldman or one of the other owners.

Reply:
Yes, that makes sense. Changed it to just `/Tulsi.xcodeproj/`